_plugins/jekyll_get.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+require 'hash-joiner'
+require 'open-uri'
+
+module Jekyll_Get
+  class Generator < Jekyll::Generator
+    safe true
+    priority :highest
+
+    def generate(site)
+      config = site.config['jekyll_get']
+      if !config
+        return
+      end
+      if !config.kind_of?(Array)
+        config = [config]
+      end
+      config.each do |d|
+        begin
+          target = site.data[d['data']]
+          source = JSON.load(open(d['json']))
+          if target
+            HashJoiner.deep_merge target, source
```

Comment:
So there is one bit of danger here. If `target` contains any non-empty Array members that are duplicated in `source`, those duplicates will appear in `target`. Since the original intent of this plugin was merging data from Midas with data stored in the Hub, this makes sense. But keeping a full cached copy of the data in the repo is a different use case.

Hence, I'm wondering if the cached version of the data should live somewhere like `_data/cached`. Then, in this plugin, in the `!config` case, we can use: `HashJoiner.promote_data site.data, 'cached'`.